.eslintrc

```diff
@@ -3,9 +3,17 @@
     "eslint:recommended",
   ],
   "plugins": [
+    "angular",
     "html"
   ],
   "rules": {
+    "angular/directive-restrict": [
+      2,
```

Comment:
What does 2 mean?

Reply:
I couldn't find a possible explanation about this in the docs. I will look into this once again.  Thanks!

Reply:
@apb7, That's Angular version. (see [here](https://github.com/Gillespie59/eslint-plugin-angular#rules-specific-for-angular-1-or-2))

Reply:
@DubeySandeep: Thank you!

Reply:
@seanlip: (Update) 
2 does not stand for the angular version, it stands for the severity:  
>Severity should be one of the following: 0 = off, 1 = warn, 2 = error`  

The link provided highlights the angular version here:  
>settings:  
    angular: 2

and not with the rule.  
Thanks!

Reply:
Noted. Thanks!

If there's a way to add a comment to the file to explain that, that would be great; if not, don't worry.

Reply:
There's an alternative to it: We can directly use "error" in place of 2. That works in the same way.

Reply:
That's great! That would be a lot better, and will avoid ambiguity. Thanks!

Reply:
Thank you @seanlip!